Add latest minor updates to Dyno RPG (#76)
DynoCC/Dyno-Custom-Commands: Fun Commands/Mini-Games/Dyno-RPG/README.md

- `sacrificial-offerings`
- `divine-council`

### How to
A simple guide for setting up roles and channels for the commands to work as intended

***The `Adventurer` role is the base role for playing Dyno RPG. As such it is suggested to use the role selection menu when adding thse commands to make the commands require it.***

***Deathmatch Commands should be allowed only in `the-deathpits`***

> ***Divine Council Commands should be limited to serverMod only and only allowed to be used in `divine-council`, while the response can preferaly be set to world-gates` if desired***

Reviewer: Spelling
```suggestion
***Divine Council Commands should be limited to serverMod only and only allowed to be used in `divine-council`, while the response can preferably be set to world-gates` if desired***
```

***Plaugue and Fester should only be useable if the person calling them is `Infected`***

> ***Divine Council is not actually a role, it's a nickname for serverMods. as such I decided not to make a seperate role for it.***

Reviewer: Grammar & spelling issues
```suggestion
***Divine Council is not actually a role, it's a nickname for serverMods. As such I decided not to make a separate role for it.***
```

> ***The `rpg-cmds` command should be accessible to `Adveturer` roles only, while the `register` command should be acessible to everyone in th server so people can join Avelon***

Reviewer: Spelling & typo
```suggestion
***The `rpg-cmds` command should be accessible to `Adventurer` roles only, while the `register` command should be accessible to everyone in the server so people can join Avelon***
```

> ***All commands should preferrably also be set to both silent and delete so as to prevent anyone from knowing who is doing what***

Reviewer: Spelling
```suggestion
***All commands should preferably also be set to both silent and delete so as to prevent anyone from knowing who is doing what***
```

### Notes

- It is suggested that the `divine-council` channel be useable **ONLY** by serverMods as it is the channel required for using the Divine Council commands.

> - When creating the roles for the commands, make sure to determin which roles have which effects. The effects will be entirely up to the ones setting it up, so know that this RPG is best when each role has a specific effect on the one carrying it.

Reviewer: Spelling
```suggestion
- When creating the roles for the commands, make sure to determine which roles have which effects. The effects will be entirely up to the ones setting it up, so know that this RPG is best when each role has a specific effect on the one carrying it.
```

> - The `Adventurer` role is the base role for playing Dyno RPG. As such it is suggested to use the role selection menu when adding thse commands to make the commands require it.

Reviewer: Spelling
```suggestion
- The `Adventurer` role is the base role for playing Dyno RPG. As such it is suggested to use the role selection menu when adding these commands to make the commands require it.
```
